source/views/calendar/event-detail.js

```diff
@@ -34,6 +36,34 @@ function MaybeSection({header, content}: {header: string, content: string}) {
     : null
 }
 
+function Links({header, content}: {header: string, content: string}) {
+  const rawUrls = getUrls(content)
+  const links = Array.from(rawUrls)
```

Comment:
Would you join those two lines, please? Just inline the rawUrls variable into Array.from for me.